example/src/bin/readme.rs

```diff
@@ -17,6 +17,13 @@ pub async fn main() -> Result<(), MomentoError> {
 
     cache_client.create_cache(CACHE_NAME.to_string()).await?;
 
+    cache_client.flush_cache(CACHE_NAME).await?;
+
+    match cache_client.list_caches().await {
```

Comment:
FYI - this readme.ts file gets sucked into the README template, so we want to keep it super short. If you want to make a copy of this file and call it `cache_operations.rs` or something, and keep the `readme.ts` file short like it was before, that would work!

Reply:
was planning on reverting the changes in `readme.ts` before taking this PR out of draft, just committed it for review purposes for now 👍

Reply:
gotcha. very helpful for review ty